db/migrate/20170313132656_remove_document_type_field_from_specialist_document_editions.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveDocumentTypeFieldFromSpecialistDocumentEditions < Mongoid::Migration
+  def self.up
+    SpecialistDocumentEdition.collection.update(
```

Comment:
`SpecialistDocumentEdition` has been renamed to `SectionEdition` so you'll need to rename this for the migration to run.